File: sensor-server/src/main/kotlin/pd/reactive/sensor/server/repository/SensorData.kt
package pd.reactive.sensor.server.repository

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table
import java.time.Instant

@Table("sensor_data")
data class SensorData(
    val temperature: Int,
    val location: String,
    val instant: Instant,
    @Id var id: String? = null
)

File: reactive-web/src/domain/SensorData.ts
class SensorData {
    temperature: number;
    location: string;
    instant: Date;
    id?: string;

    constructor(temperature: number, location: string, instant: Date, id?: string) {
        this.temperature = temperature;
        this.location = location;
        this.instant = instant;
        this.id = id;
    }
}

export default SensorData